packages/neuron-wallet/src/services/sync/get-blocks.ts

```diff
@@ -60,6 +60,20 @@ export default class GetBlocks {
               input.lock = previousOutput.lock
               input.lockHash = LockUtils.lockScriptToHash(input.lock)
               input.capacity = previousOutput.capacity
+
+              if (
+                previousOutput.type &&
+                LockUtils.computeScriptHash(previousOutput.type) === daoScriptHash &&
+                previousTx.outputsData![+input.previousOutput!.index] === '0x0000000000000000'
+              ) {
+                const output = tx.outputs!.filter(o => o.daoData && o.daoData !== '0x0000000000000000')[0]
```

Comment:
a transaction may have multiple dao cell, why just pick first?

Reply:
Not sure, can withdraw step1 has two inputs of depositOutPoint and two outputs, how to check which input is the output's input, depend on header ?

Reply:
> For a Nervos DAO deposit cell at input index , a Nervos DAO withdrawing cell should be created at output index

maybe help

Reply:
Fixed.

Reply:
LGTM